README.md

```diff
@@ -277,15 +277,15 @@ For documentation building:
 `p4c` also depends on Google Protocol Buffers (Protobuf). `p4c` requires version
 3.0 or higher, so the packaged version provided in Ubuntu 20.04 **should**
 work. However, P4C typically installs its own version of Protobuf using CMake's `FetchContent` module
-(at the moment, 3.25.3). If you are experiencing issues with the Protobuf version shipped with your OS distribution, we recommend that to install Protobuf 3.25.3 from source. You can find instructions
+(at the moment, 3.20.2). If you are experiencing issues with the Protobuf version shipped with your OS distribution, we recommend that to install Protobuf 3.20.2 from source. You can find instructions
 [here](https://github.com/protocolbuffers/protobuf/blob/v3.25.3/src/README.md).
```

Comment:
Description mentions [ v 3.20.2 ], but the guide provided is for v 3.25.3 of Protobuf

Reply:
3.25.3 is the correct one